Tests.Integration.Presentation.Web/SystemUsage/V2/ItSystemUsageApiV2Test.cs

```diff
@@ -506,6 +506,29 @@ public async Task Can_Get_Specific_ItSystemUsage_Permissions(OrganizationRole us
             Assert.Equivalent(expected, permissionsResponseDto);
         }
 
+        [Theory]
+        [InlineData(OrganizationRole.LocalAdmin, true)]
+        [InlineData(OrganizationRole.User, false)]
+        public async Task Can_Get_ItSystemUsage_CollectionPermissions(OrganizationRole userRole, bool expectCreate)
+        {
+            //Arrange
+            var (token, user, organization, system) = await CreatePrerequisitesAsync();
+            var organization2 = await CreateOrganizationAsync(A<OrganizationTypeKeys>());
+
+            await HttpApi.SendAssignRoleToUserAsync(user.Id, userRole, organization2.Id).DisposeAsync();
+            await ItSystemHelper.TakeIntoUseAsync(system.Id, organization.Id);
```

Comment:
why are you taking the system into use??? Copy-paste?

Reply:
I actually thought that it's necessary for the system to be "in use", so that it creates a System Usage

Reply:
no the "create" permission on the "it-system-usage" permission is no different from the others. It answers the question "can I create a new it-system-usage"

Reply:
but it's the permissions branch, so it's not necessary